index.bs

```diff
@@ -1566,12 +1566,35 @@ for each [=sensor types=]:
     especially when doing so would not make sense.
 -   A set of [=identifying parameters=]. TODO: replace that by an abstract operation.
 
-
 <h3 id="permission-api">Extending the Permission API</h3>
 
-Provide guidance on how to extend the Permission API [[PERMISSIONS]]
-for each [=sensor types=].
+{{Sensor}} interface for concrete [=sensor=] must protect it's [=sensor reading|reading=]
+by associated {{PermissionName}} or more complex {{PermissionDescriptor}}.
+[=Low-level=] {{Sensor|sensor}} may use it's interface name as a {{PermissionName}}, for instance
+"gyroscope" or "accelerometer". [=sensor fusion|Fusion sensors=] must
+[=request permission to use|request permission to use=] sensors used as a source of fusion.
```

Comment:
request permissions for each of the fused sensors ?

Reply:
This might sound that source sensors are also fusion sensors.